web/styles/inputs.css

```diff
@@ -0,0 +1,86 @@
+.input-element-wrapper {
+    display: grid;
+    grid-template: "icon input button" auto / auto minmax(0, 1fr) auto;
+    grid-template-rows: 1fr;
+    align-items: center;
+}
+
+.input-icon {
+    pointer-events: none;
+    grid-area: icon;
+    margin-left: 0.5em;
+    color: var(--color-text-input);
+    z-index: 1;
+}
+
+.input-active-styles {
+    color: var(--color-text-input-focus);
+    background-color: var(--color-background-input-focus);
+    border-color: var(--color-border-input-focus);
+}
+
+.input-element-wrapper .input-element {
+    grid-column: icon-start / button-end;
+    grid-row: 1;
+    box-sizing: border-box;
+    border: 1px solid var(--color-border-input);
+    padding: 0.1875em 0.5em;
+    outline: none;
+    background: var(--color-background-input);
+    font-family: "Source Sans 3 VF", sans-serif;
+    font-size: var(--base-font-size-px);
+    line-height: 1.25;
```

Comment:
Should this track to the user-set line-height? (Probably worth a design discussion.) If `1.25` is a minimum acceptable value, then something like `max()` might be useful here along with the user-set line-height variable.

Reply:
It would also be nice to better organize the styles in this block. They jump around all over. Grouping box/grid stuff, color stuff, and typography stuff would be a step in the right direction.

Reply:
I have organised and grouped the styles in this block properly. Changing the line-height will change the height of this component and would require some adjustments from my end, and in the UI region that it is situated in doesn't seem to react to the changes in line-height at the moment.